Feature/update databricks getting started (#1433)
dbt-labs/docs.getdbt.com: website/snippets/tutorial-initiate-project.md

> Now that we have a repository configured, let's initialize your project and start development in the dbt Cloud IDE:

Reviewer: Oops I missed this one previously but let's use 2nd person:
```suggestion
Now that you have a repository configured, you can initialize your project and start development in the dbt Cloud IDE:
```
